Gruntfile.js

```diff
@@ -100,17 +116,17 @@ module.exports = function (grunt) {
           port: 9001,
           middleware: function (connect) {
             return [
-              connect.static('.tmp'),
+              connect.static('<%= yeoman.stage %>/'),
               connect.static('test'),
               connect().use(
-                '/bower_components',
-                connect.static('./bower_components')
+                '/vendor',
```

Comment:
Reasonably sure that connect:test does not run minification first.  It only appears in the path of test:client and then only because of the autoprefixer writing the end-of-the-chain copy of main.css there.

And yes, I agree that <%= yeoman.vendor %> would be more appropriate here.  That change is included here now.